ITEST/itest.healthchecks/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.openl.itest</groupId>
+        <artifactId>itest</artifactId>
+        <version>5.24.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>itest.healthchecks</artifactId>
+    <packaging>jar</packaging>
+    <name>ITEST - RuleService Healthchecks</name>
+    <url>https://openl-tablets.org</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.openl.itest</groupId>
+            <artifactId>server-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+</project>
```

Comment:
Where EOL symbol in the end of the file?

Reply:
It was added now, but from GitHub UI in the browser, it's not shown. It could be checked by clicking on edit (https://github.com/openl-tablets/openl-tablets/blob/EPBDS-11111/ITEST/itest.healthchecks/pom.xml)